src/Http/Http/src/Features/FormFeature.cs

```diff
@@ -276,7 +276,9 @@ private async Task<IFormCollection> InnerReadFormAsync(CancellationToken cancell
         private static Encoding FilterEncoding(Encoding? encoding)
         {
             // UTF-7 is insecure and should not be honored. UTF-8 will succeed for most cases.
+#pragma warning disable CS0618, MSLIB0001 // Type or member is obsolete
```

Comment:
@GrabYourPitchforks the compiler \ IDE do not refer to the `MSLIB0001` diagnostic code when reporting this as being obsolete. I continued getting `CS0618` errors after it was suppressed.